src/components/auth/PrivacyAwareSignInButton.tsx

```
@@ -0,0 +1,122 @@
+}: PrivacyAwareSignInButtonProps) {
+  const [showPrivacyConsent, setShowPrivacyConsent] = useState(false);
+  const { openSignIn, openSignUp } = useClerk();
+
+  const handleSignInClick = () => {
+    if (mode === 'modal') {
+      // Show privacy consent first for new users
+      setShowPrivacyConsent(true);
+    }
+  };
+
+  const handlePrivacyConsentComplete = (hasConsent: boolean) => {
```

ESLint

[error] 'hasConsent' is defined but never used.

(@typescript-eslint/no-unused-vars)